app/javascript/app/utils/ms-formatter.js

```diff
@@ -1,18 +1,24 @@
-function formatMinutes(minutes) {
-  return minutes || 0;
-}
-
-function formatSeconds(seconds) {
-  return seconds < 10 ? `0${seconds}` : seconds;
-}
+const { I18n } = window.LoginGov;
+const formatTime = (time, unit) =>
+  // i18n-tasks-use t('datetime.dotiw.seconds.one')
+  // i18n-tasks-use t('datetime.dotiw.seconds.other')
+  // i18n-tasks-use t('datetime.dotiw.minutes.one')
+  // i18n-tasks-use t('datetime.dotiw.minutes.other')
+  time === 1
+    ? I18n.t(`datetime.dotiw.${unit}.one`)
+    : I18n.t(`datetime.dotiw.${unit}.other`, { count: time });
 
-export default (milliseconds) => {
+export function msFormatter(milliseconds) {
   const seconds = milliseconds / 1000;
   const minutes = parseInt(seconds / 60, 10);
   const remainingSeconds = parseInt(seconds % 60, 10);
 
-  const displayMinutes = formatMinutes(minutes);
-  const displaySeconds = formatSeconds(remainingSeconds);
+  const displayMinutes = formatTime(minutes, 'minutes');
+  const displaySeconds = formatTime(remainingSeconds, 'seconds');
 
-  return `${displayMinutes}:${displaySeconds}`;
-};
+  const displayTime = `${displayMinutes} ${I18n.t(
+    'datetime.dotiw.two_words_connector',
+  )} ${displaySeconds}`;
```

Comment:
The connector already includes the spaces, so those should be omitted here to avoid doubling up.

https://github.com/18F/identity-idp/blob/f56b22a963888118aa87f5f35833d3445ad546c5/config/locales/datetime/en.yml#L11

```suggestion
  const displayTime = `${displayMinutes}${I18n.t(
    'datetime.dotiw.two_words_connector',
  )}${displaySeconds}`;
```

Or:

```suggestion
  const displayTime = [
    displayMinutes,
    I18n.t('datetime.dotiw.two_words_connector'),
    displaySeconds,
  ].join('');
```

Reply:
The issue is that the I18n javascript code seems to strip off the extra spaces while the ruby `t()` doesn't. When I remove the spaces here we get `XX minutesandXX seconds` hence the need for the extra spaces.